### DISCUSSION TITLE
I can't get user on _app.js

### DISCUSSION THREAD

**Discussion by summerisgood1** [gladly-team/next-firebase-auth#204] [Q&A]
My code:
![Screenshot 2021-06-10 at 10 38 46 AM](https://user-images.githubusercontent.com/51513183/121456084-0fcacd00-c9d8-11eb-9496-9e3af43b08a9.png)

Output on console
![Screenshot 2021-06-10 at 10 38 24 AM](https://user-images.githubusercontent.com/51513183/121456094-12c5bd80-c9d8-11eb-8ba6-ff297e372e1c.png)

**Comment by moises-marquez** [comment]
Are you wrapping `_app.js` with the `withAuthUser` HOC?
